### PR TITLE
Remove test helpers

### PR DESCRIPTION
**Pull request by FK78**
Resolves #3264 

**Overall change:** _Exported @bbc/psammead-test-helpers into Simorgh._

**Code changes:**

- _Removed testHelpers.js._
- _Changed import statements to use @bbc/psammead-test-helpers._
- _Updated snapshots._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/testHelpers/loggerMock.js
Comment (dr3) [inline src/testHelpers/loggerMock.js]:
```suggestion
const mocks = {
```

File: src/server/utilities/logResponseTime/index.test.js
Comment (dr3) [inline src/server/utilities/logResponseTime/index.test.js]: Can you change these to `import loggerMock from`... everywhere please :)

**Review by dr3 — CHANGES_REQUESTED** [review]

File: src/app/components/IfAboveIE9Comment/__snapshots__/index.test.jsx.snap
Comment (dr3) [inline src/app/components/IfAboveIE9Comment/__snapshots__/index.test.jsx.snap]: Snapshot no longer reflects previous coverage, needs investigating

File: src/app/components/Metadata/helpers/__snapshots__/iconLinks.test.js.snap
Comment (dr3) [inline src/app/components/Metadata/helpers/__snapshots__/iconLinks.test.js.snap]: Snapshot no longer reflects previous coverage, needs investigating

File: src/app/components/ResourceHints/__snapshots__/index.test.jsx.snap
Comment (dr3) [inline src/app/components/ResourceHints/__snapshots__/index.test.jsx.snap]: Snapshot no longer reflects previous coverage, needs investigating

File: src/app/containers/ATIAnalytics/canonical/__snapshots__/index.test.jsx.snap
Comment (dr3) [inline src/app/containers/ATIAnalytics/canonical/__snapshots__/index.test.jsx.snap:5]: Snapshot no longer reflects previous coverage, needs investigating

File: src/app/containers/ArticleFigure/__snapshots__/index.test.jsx.snap
Comment (dr3) [inline src/app/containers/ArticleFigure/__snapshots__/index.test.jsx.snap:103]: Snapshot no longer reflects previous coverage, needs investigating

File: src/app/containers/ImageWithPlaceholder/__snapshots__/index.test.jsx.snap
Comment (dr3) [inline src/app/containers/ImageWithPlaceholder/__snapshots__/index.test.jsx.snap:46]: Snapshot no longer reflects previous coverage, needs investigating

File: src/app/containers/PageHandlers/withPageWrapper/__snapshots__/index.test.jsx.snap
Comment (dr3) [inline src/app/containers/PageHandlers/withPageWrapper/__snapshots__/index.test.jsx.snap]: Snapshot no longer reflects previous coverage, needs investigating

**Review by dr3 — CHANGES_REQUESTED** [review]
Code looks good, but some of the new snapshots dont reflect the info in the old ones, need to investigate why and how we could bring it back :)

**Comment by FK78** [comment]
Blocked by #[1917](https://github.com/bbc/psammead/pull/1917/files). Closed till further notice.

File: src/app/components/IfAboveIE9Comment/__snapshots__/index.test.jsx.snap
Comment (jroebu14) [inline src/app/components/IfAboveIE9Comment/__snapshots__/index.test.jsx.snap]: Timestamp text here is a little bit misleading https://github.com/bbc/simorgh/blob/6586a197a668233cf49b30df7ecd506396e6dc02/src/app/components/IfAboveIE9Comment/index.test.jsx#L5

**Comment by FK78** [comment]
Snapshot coverage issue may be resolved by PR #[1958](https://github.com/bbc/psammead/pull/1958)

File: src/app/components/Metadata/__snapshots__/index.test.jsx.snap
Comment (FK78) [inline src/app/components/Metadata/__snapshots__/index.test.jsx.snap]: Snapshot coverage may be restored after PR #[3694](https://github.com/bbc/simorgh/pull/3694)

File: src/app/components/LinkedData/__snapshots__/index.test.jsx.snap
Comment (FK78) [inline src/app/components/LinkedData/__snapshots__/index.test.jsx.snap:3]: Once PR #3694 is merged, snapshot coverage might be fixed.

File: src/app/components/Metadata/__snapshots__/index.test.jsx.snap
Comment (FK78) [inline src/app/components/Metadata/__snapshots__/index.test.jsx.snap:3]: Once PR #3694 is merged, snapshot coverage might be fixed.

File: src/app/Layouts/defaultPageWrapper.test.jsx
Comment (dr3) [inline src/app/Layouts/defaultPageWrapper.test.jsx:10]: With this change you can delete
```
<GlobalStyle />
<ServiceWorkerContainer />
<ManifestContainer />
<MPulseBeacon />
```
In `simorgh/src/app/Layouts/defaultPageWrapper.jsx` and it still passes, can you add assertion tests to ensure we maintain coverage please?

File: src/app/components/IfAboveIE9Comment/index.test.jsx
Comment (dr3) [inline src/app/components/IfAboveIE9Comment/index.test.jsx]:
```suggestion
describe('IfAboveIE9Comment', () => {
```

File: src/app/components/LinkedData/__snapshots__/index.test.jsx.snap
Comment (dr3) [reply src/app/components/LinkedData/__snapshots__/index.test.jsx.snap:3]: We should block this PR on that then :), lots of lost coverage

File: src/app/containers/ATIAnalytics/canonical/__snapshots__/index.test.jsx.snap
Comment (dr3) [inline src/app/containers/ATIAnalytics/canonical/__snapshots__/index.test.jsx.snap:3]: Im not sure this is working, whats the `undefined`?

File: src/app/containers/ATIAnalytics/canonical/index.test.jsx
Comment (dr3) [inline src/app/containers/ATIAnalytics/canonical/index.test.jsx:32]: Not sure this key is right, i think it needs to be something that `render` provides?

File: src/app/containers/ATIAnalytics/canonical/index.test.jsx
Comment (dr3) [inline src/app/containers/ATIAnalytics/canonical/index.test.jsx:37]: Whats this doing? Im not sure these are the same

File: src/app/containers/App/App.test.jsx
Comment (dr3) [inline src/app/containers/App/App.test.jsx:60]: Commented out?

File: src/app/containers/App/index.jsx
Comment (dr3) [inline src/app/containers/App/index.jsx:26]: Whats this?

File: src/app/containers/App/__snapshots__/index.test.jsx.snap
Comment (dr3) [inline src/app/containers/App/__snapshots__/index.test.jsx.snap:3]: File deleted?

File: src/app/containers/App/index.test.jsx
Comment (dr3) [inline src/app/containers/App/index.test.jsx:30]: We need to find an alternative to `expect.anything()`, happy to help find one :)

File: src/app/containers/ArticleFigure/__snapshots__/index.test.jsx.snap
Comment (dr3) [reply src/app/containers/ArticleFigure/__snapshots__/index.test.jsx.snap:103]: +1

File: src/app/containers/Blocks/index.test.jsx
Comment (dr3) [inline src/app/containers/Blocks/index.test.jsx:78]: Should this be a `isNull`?

File: src/app/containers/Blocks/__snapshots__/index.test.jsx.snap
Comment (dr3) [inline src/app/containers/Blocks/__snapshots__/index.test.jsx.snap:10]: Lots of stuff missing here, any ideas why Its not rendering stuff like `This is a headline!`?

File: src/app/containers/Footer/__snapshots__/index.test.jsx.snap
Comment (dr3) [inline src/app/containers/Footer/__snapshots__/index.test.jsx.snap:314]: Does this need an isNull check?

**Review by dr3 — CHANGES_REQUESTED** [review]

File: src/app/containers/App/index.jsx
Comment (FK78) [reply src/app/containers/App/index.jsx:26]: Notes I've left in by accident.

**Comment by FK78** [comment]
PR is being split up so it is easier to review.

### Small changes
- #3708
- #3776
- #3814

### Major changes
- #3711
- #3842
- #3854
- #3886
- #3945
- #4058

**Pull request closed without merging**
